tools/version.props.template

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <PropertyGroup>
+    <Version>@Version</Version>
```

Comment:
The `@Version` tag is replaced by the contents of the `.version` file during the build process.